app/components/primer/tooltip.rb

```diff
@@ -70,7 +75,7 @@ def initialize(
       **system_arguments
     )
       @system_arguments = system_arguments
-      @system_arguments[:tag] ||= :span # rubocop:disable Primer/NoTagMemoize
+      @system_arguments[:tag] = fetch_or_fallback(TAG_OPTIONS, tag, TAG_DEFAULT)
```

Comment:
@manuelpuyol 

On dotcom I see a lot of something like:
`<button class="btn disabled tooltipped tooltipped-n"`  (and a whole bunch of other elements).

Is [the wrapping syntax](https://primer.style/view-components/components/tooltip#wrapping-another-component) a direct replacement?

Reply:
I think this would be a case-by-case scenario, since the wrapped element would decide the `tag` for the tooltip wouldn't it?